dashboard/app/views/devise/registrations/edit.html.haml

```diff
@@ -5,6 +5,7 @@
 - require 'queries/user'
 - @page_title = t('activerecord.attributes.user.edit_header')
 - no_email = current_user.no_personal_email? || params[:noEmail] == 'true'
+- studentInLockoutFlow = Policies::ChildAccount.underage?(current_user) && !Policies::ChildAccount::ComplianceState.permission_granted?(current_user)
```

Comment:
```suggestion
- studentInLockoutFlow = !!Policies::ChildAccount.lockout_date(user)
```
Alternatively, I think this method does the same thing. It should return the date in which a user will be locked out, which is essentially the same as determining that they are in the "grace_period".

You should double-check this suggestion by making sure your UI test still passes.
 
One use case to consider is `U13 + Colorado + Clever`. I don't think we want to lock the age and state, right?

Reply:
Correct, we are not locking out age and state for clever users.

Reply:
During the bugbash, it was confirmed that we want to prevent U13 + CO + LTI users from editing age and state. If we consider LTI and Clever to be in the same bucket, wouldn't we want to lock out age and state for Clever users?

Reply:
> During the bugbash, it was confirmed that we want to prevent U13 + CO + LTI users from editing age and state. If we consider LTI and Clever to be in the same bucket, wouldn't we want to lock out age and state for Clever users?

Yeah you are right @carl-codeorg, I was wrong about this feedback.